restructure Search content (#3701)
getsentry/sentry-docs: src/docs/product/sentry-basics/search/saved-searches.mdx

---
title: Saved Searches
sidebar_order: 20
keywords: ["saved search", "premade searches"]
description: "Learn more about recommended, pinned, and saved searches."
---

In the "Saved Searches" tab of **Issues**, you can access recommended searches, pin searches that you find most useful, and take advantage of organization-wide saved searches.

## Recommended Searches

Recommended searches are common search terms that we think you're likely to use. These premade searches are listed under "Recommended Searches" in the "Saved Searches" dropdown, and they're listed in order of when you've most recently used them.

![Open Saved Searches dropdown on the Issues page.](search-premade-dropdown.png)

## Pinned Searches

> You can pin a search, and it will become the default view you see on the **Issues** page. The pinned search is only visible to you and is relevant across your projects.

Reviewer: ```suggestion
You can pin a search, and it will become the default view you see on the **Issues** page. The pinned search is visible only to you and is relevant across your projects.
```

> 1. Type a search into the search bar.

Reviewer: ```suggestion
1. Type search terms into the search bar.
```

   ![The search bar of the Issues page with two search terms.](search-pinning-step-one.png)

2. Click the pin icon next to that search.

   ![Cursor mousing over the pin icon and tool-tip displayed.](search-pinning-step-two.png)

3. Once pinned, Sentry will add the search to the "Saved Searches" dropdown. The search label in the text will read, "My Pinned Search".

   ![Pinned search in Saved Searches dropdown.](search-pinning-step-three.png)

### Changing a Pinned Search

> To change your pinned search, do the following:

Reviewer: ```suggestion
To change your pinned search:
```

1. Select your pinned search. Un-click the pin icon. Your default search will return to `is:unresolved`.

> 2. Run another search. Click the pin icon. The query listed as "My Pinned Search" will now be the new pinned query, instead of the original one.

Reviewer: ```suggestion
2. Run another search. Click the pin icon. The query listed as "My Pinned Search" will now be the new pinned query, replacing the original one.
```

### Pinning a Recommended Search

You can pin a recommended search the same way you pin any other search. When you've selected a recommended search, and the recommended search query populates the search bar, pin it.

## Organization Wide Saved Searches

> 

Reviewer: does the lack of text here cause issues with the RH display?

Author: No, it doesn't cause issues--that's the way it is in the current docs and it renders fine. In my experience, it's just skipped heading levels that cause rendering issues.

> ### Creating an org-wide saved search

Reviewer: ```suggestion
### Creating an Organization-Wide Saved Search
```

> Owners and managers can create a persistent view for their organization by creating custom saved searched. These saved searches are not associated with a specific project, but with all projects (and users) across the org.

Reviewer: ```suggestion
Owners and managers can create a persistent view for their organization by creating custom saved searched. These saved searches are not associated with a specific project, but with all projects (and users) across the organization.
```

1. Type a search into the search bar, click the actions menu (three dots). Select "Create Saved Search".

   ![Cursor mousing over Create Saved Search in actions menu.](search-add-org-filter-step-one.png)

2. In the modal that opens, name the search and set the sort order for the issues list. You can also update the query here. Then click "Save".

   ![Modal for naming and setting sort order of search.](search-add-org-filter-step-two.png)

3. The view will then become part of the "Saved Search" dropdown.

   ![Saved search in the dropdown.](search-add-org-filter-step-three.png)

> ### Deleting an org-wide saved search

Reviewer: ```suggestion
### Deleting an Organization-Wide Saved Search
```

> 

Author: ```suggestion

<Note>

This action is only available for organization owners or managers.

</Note>

```

> When you hover over a custom saved search, you can see a trash icon next to the name of the search. Click the trash icon to remove the custom saved search from the dropdown.

Reviewer: ```suggestion
When you hover over a custom saved search, the trash icon displays next to the name of the search. Click the trash icon to remove the custom saved search from the dropdown.
```

> This action is only available for organization owners or managers.

Reviewer: s/be a note? If so, place it above line 64.

> This action is only available for organization owners or managers.

Author: ```suggestion
```
